src/Misc/externals.sh

```diff
@@ -11,6 +11,7 @@ NODE_URL=https://nodejs.org/dist
 NODE_VERSION="6.17.1"
 NODE10_VERSION="10.24.1"
 NODE16_VERSION="16.17.1"
+NODE20_VERSION="20.0.0"
```

Comment:
The latest node 20 version is now 20.3.1: https://nodejs.org/en/download/current